release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -480,6 +480,24 @@ xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k
 Configuration must occur before scheduling pods.
 ====
 
+[id="ocp-4-7-network-policy-host-network-ingress-controllers"]
+==== Network policy supports selecting host network Ingress Controllers
+
+When using the OpenShift SDN or OVN-Kubernetes cluster network providers, you can select traffic from Ingress Controllers in a network policy rule regardless of whether an Ingress Controller runs on the cluster network or the host network.
+In a network policy rule, the `network.openshift.io/policy-group: ingress` namespace selector matches traffic from an Ingress Controller.
+
+In earlier releases of {product-title} the following limitations existed:
+
+- A cluster using the OpenShift SDN cluster network provider could select traffic from an Ingress Controller on the host network only by applying the `network.openshift.io/policy-group: ingress` label to the `default` namespace.
```

Comment:
```suggestion
- A cluster that uses the OpenShift SDN cluster network provider could select traffic from an Ingress Controller on the host network only by applying the `network.openshift.io/policy-group: ingress` label to the `default` namespace.
```

> ! using noun
To avoid ambiguity, replace this gerund with either "by using" or "that uses".

Also, remember that this is one of the only acceptable uses of "could."